libs/client/cornflow_client/constants.py

```diff
@@ -90,6 +90,74 @@
     pl.LpStatusUndefined: STATUS_UNDEFINED,
 }
 
+PARAMETER_SOLVER_TRANSLATING_MAPPING = {
+    ('bar_iter_limit', 'pyomo', 'cbc') : 'barr' ,
+    ('best_obj_stop', 'pyomo', 'cbc') : 'primalT' ,
+    ('cutoff', 'pyomo', 'cbc') : 'cuts' ,
+    ('iteration_limit', 'pyomo', 'cbc') : 'maxIt' ,
+    ('solution_limit', 'pyomo', 'cbc') : 'maxSaved' ,
+    ('time_limit', 'pyomo', 'cbc') : 'sec' ,
+    ('mip_gap', 'pyomo', 'cbc') : 'allow' ,
+    ('optimality_tol', 'pyomo', 'cbc') : 'dualT' ,
+    ('pump_passes', 'pyomo', 'cbc') : 'pumpC' ,
+    ('heuristics', 'pyomo', 'cbc') : 'heur' ,
+
+    ('is_mip', 'pulp', 'cbc') : 'mip' ,
+    ('optimality_tol', 'pulp', 'cbc') : 'gapAbs' ,
+    ('cutoff', 'pulp', 'cbc') : 'cuts' ,
+    ('time_limit', 'pulp', 'cbc') : 'timeLimit' ,
+    ('threads', 'pulp', 'cbc') : 'threads' ,
+    ('presolve', 'pulp', 'cbc') : 'presolve' ,
+
+    ('bar_iter_limit', 'pyomo', 'gurobi') : 'BarIterLimit' ,
+    ('best_obj_stop', 'pyomo', 'gurobi') : 'BestObjStop' ,
+    ('cutoff', 'pyomo', 'gurobi') : 'Cutoff' ,
+    ('iteration_limit', 'pyomo', 'gurobi') : 'IterationLimit' ,
+    ('mem_limit', 'pyomo', 'gurobi') : 'MemLimit' ,
+    ('solution_limit', 'pyomo', 'gurobi') : 'SolutionLimit' ,
+    ('time_limit', 'pyomo', 'gurobi') : 'TimeLimit' ,
+    ('mip_gap', 'pyomo', 'gurobi') : 'MIPGap' ,
+    ('optimality_tol', 'pyomo', 'gurobi') : 'OptimalityTol' ,
+    ('branch_dir', 'pyomo', 'gurobi') : 'BranchDir' ,
+    ('pump_passes', 'pyomo', 'gurobi') : 'PumpPasses' ,
+    ('heuristics', 'pyomo', 'gurobi') : 'Heuristics' ,
+    ('presolve', 'pyomo', 'gurobi') : 'Presolve' ,
+    ('threads', 'pyomo', 'gurobi') : 'threads' ,
+
+    ('is_mip', 'pulp', 'gurobi') : 'mip' ,
+    ('time_limit', 'pulp', 'gurobi') : 'timeLimit' ,
```

Comment:
Same here, how do you set up the gap for gurobi?

Reply:
In this two lines, the gap is set
    ('mip_gap', 'pyomo', 'gurobi') : 'MIPGap' ,
    ('optimality_tol', 'pyomo', 'gurobi') : 'OptimalityTol' ,
These are most used options

Reply:
And in pulp?

Reply:
Unfortunately, the "direct" gurobi parameters accepted in pulp are: 

mip ([bool]) – if False, assume LP even if integer variables

msg ([bool]) – if False, no log is shown

options ([list]) –

timeLimit ([float]) – maximum time for solver (in seconds)

args –

kwargs – optional named options to pass to each solver, e.g. gapRel=0.1, gapAbs=10, logPath=””, 

Therefore, the only feasible option I see is to adapt the kwargs parameter to include in gapAbs, but should be done carefully